scenarios/classification/21_deployment_on_azure_container_instances.ipynb

```diff
@@ -30,8 +30,8 @@
     "  1. [Workspace retrieval](#workspace)\n",
```

Comment:
<p>If it's quick to do, maybe run the notebook one more time so that the outputs after each cell are part of the PR. Any cell with confidential information of course remove the output for just that cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/529/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done.